firebase-web/src/main/java/io/spine/web/firebase/NodeValue.java

```diff
@@ -69,11 +69,18 @@ public static NodeValue from(String json) {
     }
 
     /**
-     * Creates a {@code NodeValue} which has a single entry under a generated key.
+     * Creates a new node value with all the given JSONs as its children.
+     *
+     * <p>The child nodes are added under generated keys.
+     *
+     * @param jsons child nodes
+     * @return new node value
      */
-    public static NodeValue withSingleChild(String childValue) {
+    public static NodeValue withChildren(Iterable<String> jsons) {
```

Comment:
Why do we speak about parts of that Json tree as about `String`s? I don't mind having `String`s in the internal API. But this one is public.